assets/resources.qrc

```diff
@@ -1,6 +1,7 @@
 <RCC>
     <qresource prefix="/">
         <file>aa_video.qml</file>
+	    	<file>splash.png</file>
```

Comment:
change to spaces instead of tabs (to keep the spacing consistent)

Reply:
corrected tabs to spaces per comment